File: examples/metal/port/layer2_bonded/go/main.go
package main

import (
	"github.com/equinix/pulumi-equinix/sdk/go/equinix/metal"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi/config"
)

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		cfg := config.New(ctx, "")
		portId := cfg.Require("portId")
		port, err := metal.NewPort(ctx, "port", &metal.PortArgs{
			PortId: pulumi.String(portId),
			Bonded: pulumi.Bool(true),
			Layer2: pulumi.Bool(true),
		})
		if err != nil {
			return err
		}
		ctx.Export("portType", port.Type)
		ctx.Export("portBondedNetworkType", port.NetworkType)
		return nil
	})
}

File: examples/metal/port/layer2_bonded/python/__main__.py
import pulumi
import pulumi_equinix as equinix

config = pulumi.Config()
port_id = config.require("portId")
port = equinix.metal.Port("port",
    port_id=port_id,
    bonded=True,
    layer2=True)
pulumi.export("portType", port["type"])
pulumi.export("portBondedNetworkType", port["networkType"])
